stable-profiles/0.9.52/firejailed-tor-browser.profile

```diff
@@ -0,0 +1,117 @@
+whitelist /usr/share/alsa
+whitelist /usr/share/applications
+whitelist /usr/share/ca-certificates
+whitelist /usr/share/ca-certificates-java
+##whitelist /usr/share/crypto-policies
+##whitelist /usr/share/cursors
+whitelist /usr/share/dconf
+whitelist /usr/share/distro-info
+whitelist /usr/share/drirc.d
+whitelist /usr/share/enchant
+##whitelist /usr/share/enchant-2
+whitelist /usr/share/fontconfig
+whitelist /usr/share/fonts
+##whitelist /usr/share/gir-1.0
+##whitelist /usr/share/gjs-1.0
+whitelist /usr/share/glib-2.0
+whitelist /usr/share/glvnd
+##whitelist /usr/share/gtk-2.0
+##whitelist /usr/share/gtk-3.0
+whitelist /usr/share/gtksourceview-3.0
+##whitelist /usr/share/gtksourceview-4
+whitelist /usr/share/hunspell
+##whitelist /usr/share/hwdata
+whitelist /usr/share/icons
+whitelist /usr/share/knotifications5
+whitelist /usr/share/kservices5
+##whitelist /usr/share/Kvantum
+whitelist /usr/share/kxmlgui5
+whitelist /usr/share/libdrm
+whitelist /usr/share/libthai
+whitelist /usr/share/locale
+whitelist /usr/share/mime
+whitelist /usr/share/misc
+##whitelist /usr/share/Modules
+##whitelist /usr/share/myspell
+whitelist /usr/share/p11-kit
+whitelist /usr/share/pixmaps
+##whitelist /usr/share/pki
+##whitelist /usr/share/plasma
+##whitelist /usr/share/qt
+##whitelist /usr/share/qt4
+whitelist /usr/share/qt5
+whitelist /usr/share/sounds
+##whitelist /usr/share/tcl8.6
+whitelist /usr/share/tcltk/
+whitelist /usr/share/terminfo
+whitelist /usr/share/themes
+##whitelist /usr/share/thumbnail.so
+whitelist /usr/share/X11
+whitelist /usr/share/xml
+whitelist /usr/share/zoneinfo
+# Add the next two lines to firejailed-tor-browser.local to enable better desktop integration
+#include /etc/firejail/whitelist-common.inc
+#include /etc/firejail/whitelist-var-common.inc
+
+#apparmor
+caps.drop all
+netfilter
+# Disable hardware acceleration
+#no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+# Disable sound, enable if you don't need
+#nosound
+notv
+novideo
+protocol unix,inet,inet6
+#seccomp - replaced with seccomp.drop for Firefox 60
+# @default-nodebuggers without chroot
+seccomp.drop @clock,@cpu-emulation,@debug,@module,@obsolete,@raw-io,@reboot,@resources,@swap,acct,add_key,bpf,fanotify_init,io_cancel,io_destroy,io_getevents,io_setup,io_submit,ioprio_set,kcmp,keyctl,mount,name_to_handle_at,nfsservctl,open_by_handle_at,personality,pivot_root,process_vm_readv,ptrace,remap_file_pages,request_key,setdomainname,sethostname,syslog,umount2,userfaultfd,vhangup,vmsplice
+seccomp.block-secondary
+shell none
+# Cause some issues
+#tracelog
+
+disable-mnt
+private ${HOME}/.firejailed-tor-browser
+private-bin bash,sh,grep,tail,env,expr,file,getconf,gpg,id,readlink,dirname,test,mkdir,python*,ln,sed,cp,rm,start-tor-browser.desktop,start-tor-browser,firejailed-tor-browser
+#private-cache
+private-dev
+private-etc fonts
+#profile.template#L151-L162
+private-etc machine-id
+private-tmp
+
+name firejailed-tor-browser
+#noexec ${RUNUSER}
+noexec /dev/shm
+noexec /tmp
```

Comment:
Is `${RUNUSER}` not supported in 0.9.52 or why is this commented?

Reply:
Good question. Initially I did try this option a few times but I kept received the following errors:
```
Error: "${RUNUSER}" is an invalid filename
```
So I it seems that this option/feature might not be support in 0.9.52. Perhaps the statement is incorrect? I don't know. That is the reason why I have commented this option.

Reply:
Yes, it is then not supported and can be removed.

Reply:
Ok, I will remove it. Thank you.